AI/VCAI/Pathfinding/Actions/BoatActions.cpp

```diff
@@ -0,0 +1,60 @@
+/*
+* BoatActions.cpp, part of VCMI engine
+*
+* Authors: listed in file AUTHORS in main folder
+*
+* License: GNU General Public License v2.0 or later
+* Full text of license available in license.txt file, in main folder
+*
+*/
+
+#include "StdInc.h"
+#include "../../Goals/AdventureSpellCast.h"
+#include "../../Goals/BuildBoat.h"
+#include "../../../../lib/mapping/CMap.h"
+#include "../../../../lib/mapObjects/MapObjects.h"
```

Comment:
Can i reference it in some simplier way? Like #include "mapping/CMap.h". It should be already in some include path

Reply:
For now it should stay relative path.